crates/codegen/language/tests/src/fail/parsing/duplicate_map_key/test.stderr

```diff
@@ -1,5 +1,5 @@
-error: Duplicate map key.
-  --> src/fail/parsing/duplicate_map_key/test.rs:19:25
+error: unexpected token
+  --> src/fail/parsing/duplicate_map_key/test.rs:30:5
    |
-19 |                         field_1 = Required(Baz),
-   |                         ^^^^^^^
+30 |     queries = ()
```

Comment:
I think these files now have the wrong snapshots?
I suggest fixing the test by removing the now obsolete `queries` field.

Reply:
done